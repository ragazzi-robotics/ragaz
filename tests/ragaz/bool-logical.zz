def main():

    # bool
    print(True)
    print(False)

    # and/or, bool
    print(not True)
    print(True and False)
    print(True or False)

    # and/or, str
    print("a" and "b")
    print("" and "a")
    print("a" and "")
    print("a" or "b")
    print("" or "a")
    print("a" or "")
