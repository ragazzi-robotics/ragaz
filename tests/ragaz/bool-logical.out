True
False
False
False
True
b


a
a
a
